src/interfaces/assistants_web/src/components/Shared/Textarea/Textarea.tsx

```diff
@@ -0,0 +1,53 @@
+'use client';
+
+import { Field, Label } from '@headlessui/react';
+import { forwardRef } from 'react';
+
+import { STYLE_LEVEL_TO_CLASSES, Text } from '@/components/Shared/Text';
+import { cn } from '@/utils';
+
+type Props = Omit<React.HTMLProps<HTMLTextAreaElement>, 'onChange' | 'value'> & {
+  value?: string;
+  label?: string;
+  className?: string;
+  onChange?: (e: React.ChangeEvent<HTMLTextAreaElement>) => void;
+};
+
+export const Textarea: React.FC<Props> = forwardRef<HTMLTextAreaElement, Props>(
+  ({ value, label, className, onChange, ...rest }, ref) => {
+    return (
+      <Field className="flex flex-col gap-y-2">
+        {label && (
+          <Label className="flex items-start gap-x-2">
+            <Text styleAs="label" className="dark:text-marble-950">
+              {label}
+            </Text>
+          </Label>
+        )}
+        <div className="relative flex w-full items-center gap-x-2">
```

Comment:
Isn't there only one child?
```suggestion
        <div className="relative flex w-full items-center">
```

Reply:
cleaned up!